blockchain.go

```diff
@@ -1207,6 +1225,11 @@ func (b *Blockchain) ExecuteScriptAtBlock(script []byte, arguments [][]byte, blo
 	}, nil
 }
 
+func ComputeScriptID(script []byte) sdk.Identifier {
+	hasher := hash.NewSHA3_256()
+	return sdk.HashToID(hasher.ComputeHash(script))
+}
```

Comment:
Not sure, but maybe the SDK or flow-go/FVM already expose this? It would be good to have it there, so that this code stays in sync / this duplication does not get out sync.